backend/modules/email/daos/email.dao.ts

```diff
@@ -0,0 +1,98 @@
+import prismaService from "../../common/services/prisma.service";
+import debug from "debug";
+import { PrismaClient, Status } from "@prisma/client";
+import { CreateEmail } from "../dto/create.email.dto";
+import { UpdateEmail } from "../dto/update.email.dto";
+import { DeleteEmailDto, GetEmailDto, GetEmailsDto } from "../dto/get.emails.dto";
+
+const log: debug.IDebugger = debug("app:in-memory-dao");
+
+class EmailDao {
+  prisma: PrismaClient;
+
+  constructor() {
+    log("Created new instance of EmailDao");
+    this.prisma = prismaService.getPrismaClient();
+  }
+
+  async getEmailById(payload: GetEmailDto) {
+    try {
+      let {
+        id
+      } = payload;
+
+      console.log("payload before getwebsites command", payload);
+
+      return await this.prisma.email.findUnique({
+        where: {
+          id
```

Comment:
Add the userId against findUnique so when user chcanges id from url, they do not fetch the data when the userId mismatches